mobile_verifier/src/data_session.rs

```diff
@@ -146,7 +152,9 @@ pub async fn data_sessions_to_dc<'a>(
     tokio::pin!(stream);
     let mut map = HotspotMap::new();
     while let Some(session) = stream.try_next().await? {
-        *map.entry(session.pub_key).or_default() += session.num_dcs as u64
+        let rewards = map.entry(session.pub_key).or_default();
+        rewards.rewardable_dc += session.num_dcs as u64;
+        rewards.rewardable_bytes += session.upload_bytes as u64 + session.download_bytes as u64;
```

Comment:
Should we not be carrying rewardable_bytes from the proto:ValidDataTransferSession into this session object and use that to accumulate rewardable_bytes rather than the older upload and download bytes values ?

Reply:
I added rewardable_bytes to the `gateway_reward` proto, I was thinking just to make it clearer how many bytes were considered rewardable for the gateway

Reply:
further note on using rewardable_bytes vs upload + download bytes....cant the sum of upload+download bytes exceed that rewardable...hence why rewardable bytes was added ?

Reply:
It's also just more transparency - the rewardable_dc calculation uses a div_ceil operation, so the function is not exactly one to one